src/platform/activation_abi.cpp

```diff
@@ -19,17 +19,17 @@ namespace xlang::impl
             if (pfn)
             {
                 xlang_result result = (*pfn)(class_name, iid, factory);
-                if (result == xlang_error_ok)
+                if (result == xlang_result::xlang_success)
                 {
                     return nullptr;
                 }
-                else if (result != xlang_error_class_not_available)
+                else if (result != xlang_result::xlang_no_interface)
                 {
                     throw_result(result);
                 }
             }
         }
-        return originate_error(xlang_error_class_not_available);
+        return xlang_originate_error(xlang_result::xlang_no_interface);
```

Comment:
It seems like class not available should be one of the core errors listed in XDN11. I think you removed it as part of that review, but perhaps we should not have? Is it important for language projection code interacting with activation factories to distinguish between "that class doesn't exist in this binary" and "that class exists but doesn't implement the interface you asked for"?

Reply:
I looked up what we did with C++/WinRT for this and we do have a different error (returned by the OS).  I think this error maps up with the xlang_type_load error that I have in our list of error codes.  I will update the spec to call out that error for use in this scenario and and use it here.